src/EditorFeatures/Core/SolutionEvents/HostLegacySolutionEventsWorkspaceEventListener.cs

```diff
@@ -44,14 +41,11 @@ internal sealed partial class HostLegacySolutionEventsWorkspaceEventListener : I
 
     public void StartListening(Workspace workspace, object? serviceOpt)
     {
-        if (_globalOptions.GetOption(SolutionCrawlerRegistrationService.EnableSolutionCrawler))
+        workspace.WorkspaceChanged += OnWorkspaceChanged;
```

Comment:
❓ Shouldn't this method just be empty now?

Reply:
was just about to comment this.

the answer is: No.

First, the original code was wrong.  It should not have been driven off of this 'SolutionCrawler' flag at all.  

What this component is is the piece that simply notifies the external Unit Testing system that solution changes have happened.  They can then drive *their* system based on these events.  Note: longer term plan is for this to go away, and they do things in a pull/refresh fashion as well.